config-linux.md

```diff
@@ -576,7 +576,7 @@ Its value is either slave, private, or shared.
 
 ## Mount Label
 
-`mountLabel` will set the Selinux context for the mounts in the container.
+**`mountLabel`** will set the Selinux context for the mounts in the container.
```

Comment:
This (and a number of other entries you bold) should probably also get `(object, OPTIONAL)`.